### ISSUE TITLE
The module is not installable on rakudo 2019.11

### ISSUE THREAD

**Issue by Altai-man** [css-raku/CSS-raku#1]
```
===> Testing: CSS:ver<0.0.1>
[CSS] # Failed test '<i/>'
[CSS] # at t/tag-set-xhtml.t line 28
[CSS] # expected: 'font-style:italic;'
[CSS] #      got: ''
[CSS] # Failed test '<i><span/></i>'
[CSS] # at t/tag-set-xhtml.t line 30
[CSS] # expected: 'font-style:italic;'
[CSS] #      got: ''
[CSS] # You failed 2 tests of 5
===> Testing [FAIL]: CSS:ver<0.0.1>
Aborting due to test failure: CSS:ver<0.0.1> (use --force-test to override)

```

**Comment by dwarring** [comment]
my mistake. fixed in 0.0.2 uploaded to CPAN

**Comment by Altai-man** [comment]
Awesome! Thank you very much!
